### ISSUE TITLE
Preview doesn't work

### ISSUE THREAD

**Issue by RonaldBoyd** [stackblitz/webcontainer-core#1454]
### Describe the bug

The preview does not render the app at url: https://stackblitz.com/edit/stackblitz-starters-bc3yhh?file=script.js,index.html,styles.css

### Link to the blitz that caused the error

https://stackblitz.com/edit/stackblitz-starters-bc3yhh?file=script.js,index.html,styles.css

### Steps to reproduce

Open preview

### Expected behavior

App rendered in the preview

### Parity with Local

- [X] I have [run the project in my local machine](https://github.com/stackblitz/webcontainer-core/blob/main/repro.md#try-locally) and I could not reproduce the issue.

### Platform

```
Browser name  = Chrome
Full version  = 125.0.0.0
Major version = 125
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 91596649,
  "usedJSHeapSize": 86764101,
  "jsHeapSizeLimit": 2172649472
}
Hash = 12d46890
```

**Comment by SamVerschueren** [comment]
Hey @RonaldBoyd 👋 ! Thanks for reaching out.

You can read more information regarding this in the comment right [here](https://github.com/stackblitz/webcontainer-core/issues/1297#issuecomment-1894313990).

tldr; we don't automatically serve static `index.html` files. If you want to show serve the `index.html`, you could type `npx servor` which will spin up a small server and serves the static assets in your project.
